### ISSUE TITLE
BUG: Segfault in test_strtree.py::test_flush_geometries

### ISSUE THREAD

**Issue by caspervdw** [pygeos/pygeos#97]
See https://travis-ci.org/pygeos/pygeos/jobs/652638801

The master build worked. I took a diff between the dependencies in the master build and the dependencies in the link and there was only one:
```
pycparser-2.19             |             py_2          88 KB  conda-forge
```
Instead of
```
pycparser-2.19             |           py37_1         171 KB  conda-forge
```

Which seems completely unrelated, although I don't know how pycparser is used in our test suite.

**Comment by jorisvandenbossche** [comment]
Are you able to reproduce it locally with the new dependencies?

**Comment by caspervdw** [comment]
No I haven’t tried to set up the environment yet.

**Comment by brendan-ward** [comment]
@caspervdw this appears resolved by #100
